platform/android/export/export.cpp

```diff
@@ -2747,6 +2840,25 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return ERR_CANT_CREATE;
 		}
 
+		List<String> mv_args;
+		String output_path;
+		if (p_debug) {
```

Comment:
Suggest inverting this: instead of debug then aab, do aab then debug

Could also use ?: and/or share the common part of the path + concat strings